app/scripts/metamask-controller.test.js

```diff
@@ -1053,7 +1054,7 @@ describe('MetaMaskController', function () {
         streamTest.end();
       });
 
-      it('adds a tabId and origin to requests', function (done) {
+      it('adds a tabId and origin to requests, as well as networkClientId', function (done) {
```

Comment:
sorry, missed this. This reads a little funky.

`adds a tabId, origin, and networkClientId to requests`